docs/release-notes/8.0.asciidoc

```diff
@@ -8,3 +8,35 @@
 [discrete]
 [[known-issues-8.0.0-rc2]]
 ==== Known issues
+
+*The Data View option might not display in upgraded environments with legacy alerts:*
+
+NOTE: If new alerts are generated in an upgraded environment without legacy alerts, refreshing any page in {es-sec} will make the *Data View* option appear in the {es-sec-ui}.
+
+After upgrading, the *Data View* option might not display in environments with legacy alerts. To resolve this, a user with elevated role privileges must visit the {es-sec-app}, open a page that displays alert data (such as the Overview page), and refresh the page ({pull}121390[#121390]).
```

Comment:
Since both lines 14 and 16 recommend that the user refresh to make the Data View button appear, would it be more concise to combine them and just say something like "After upgrading, the *Data View* option might not appear until a user with elevated privileges visits a page that displays alerts (such as the Overview page), and refreshes the page"?

I guess that would gloss over the detail currently in line 14 that in an env without legacy alerts, you can refresh any page rather than just one that shows alerts. But would it be common for users not to have any legacy alerts?

Reply:
Thanks for this feedback! I don't know how common it'd be for users to not have legacy alerts in their environment, but that'd be the case if they started directly from `8.0` or decided to not migrate their old alerts from `7.x`. 

I'm also glad you commented on this because, as I was drafting the summary, I was wondering whether the placement of the note and the known issue description would be _too_ close. The wording in both is so similar that I could easily see a reader assuming they were the same. In reality, they describe two very different scenarios. 

**Scenario #1:**
Here, the user upgraded from `7.x` to `8.0` and, as part of the upgrade process, migrated alerts that were generated in `7.x`. Because of this, legacy alerts now exist in the upgraded environment (`8.0`) and in order to make the **Data View** option appear, a user with elevated role privs needs to visit the Security app and refresh a page that displays alert data.

**Scenario #2:**
The user did not upgrade from `7.x`, and just started using `8.0` instead. Because they didn’t upgrade from a previous version, they didn’t need to migrate any “old” alerts and the environment that they’re working in doesn’t contain legacy alerts. In this scenario, they still have to force the **Data View** option to appear, but they can only do it after new alerts are generated. Once new alerts are generated, the user can go to any page in the Security app and refresh it to force the **Data View** option to appear in the Security app.

One thing your comment made me realize was that it’s probably better to instruct the user to refresh a page that displays alert data, as opposed to any page in the Security app. That’s because, if they refresh a page that displays alert data (e.g., the Overview page), they can immediately verify that the **Data View** option is available for use. However, if they went to the Cases page and refreshed that, they wouldn’t be able to immediately see that the **Data View** option is available because the Cases page only displays case data.